scripts/cli-test.sh

```diff
@@ -25,7 +25,7 @@ start_gateway
 #### BEGIN TESTS ####
 
 # test the fetching of client configs
-CONNECT_STRING=$(cat $FM_CFG_DIR/client-connect.json)
+CONNECT_STRING=$(cat $FM_CFG_DIR/client-connect)
```

Comment:
new ending to indicate filetype?  `.bech32` or similar?

Reply:
IMO extensionless is fine. json is a well know format support by many editors, bech32 is not.

Reply:
yah, I agree. 
My thinking was more in regards to indicate what the numbers and letters are in this file. Putting `.bech32` at the end means someone does not have to look at (perhaps lacking/outdated) documentation or figure out what it is

I'm fine with either way, though